tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -55,16 +56,200 @@ def test_dates(marcxml_to_json, json_to_marc):
+                'name': 'CERN-ALPHA',
+                'record': 'mocked_recid_00001',
+                'start_year': 2010,
+            },
+        ],
+        [
+            {
+                '0': 1,
+                'd': 2020,
+                'e': 'CERN-ALPHA',
+                's': 2014,
+                'z': 'current',
+            },
+            {
+                '0': 1,
+                'd': 2012,
+                'e': 'CERN-ALPHA',
+                's': 2010,
+            },
+        ],
+    ],
+    [
+        'simultaneous_experiments',
+        '''
+        <datafield tag="693" ind1=" " ind2=" ">
+            <subfield code="d">2013</subfield>
+            <subfield code="e">FIRST-SIMULTANEOUS</subfield>
+            <subfield code="e">SECOND-SIMULTANEOUS</subfield>
+            <subfield code="0">00001</subfield>
+            <subfield code="0">00002</subfield>
+            <subfield code="s">2015</subfield>
+        </datafield>
+        ''',
+        [
+            {
+                'curated_relation': True,
+                'current': False,
+                'end_year': 2013,
+                'name': 'FIRST-SIMULTANEOUS',
+                'record': 'mocked_recid_00001',
+                'start_year': 2015,
+            },
+            {
+                'curated_relation': True,
+                'current': False,
+                'end_year': 2013,
+                'name': 'SECOND-SIMULTANEOUS',
+                'record': 'mocked_recid_00002',
+                'start_year': 2015
+            },
+        ],
+        [
+            {
+                '0': 1,
+                'd': 2013,
+                'e': 'FIRST-SIMULTANEOUS',
+                's': 2015,
+            },
+            {
+                '0': 2,
+                'd': 2013,
+                'e': 'SECOND-SIMULTANEOUS',
+                's': 2015,
+            },
+        ],
+    ],
+]
+
+
+def get_test_names():
+    for exp_data in EXPERIMENTS_DATA:
+        yield exp_data[0]
+
+
+@pytest.mark.parametrize(
+    'test_name,xml_snippet,expected_json,expected_marc',
+    EXPERIMENTS_DATA,
+    ids=list(get_test_names()),
+)
+@mock.patch('inspirehep.dojson.hepnames.fields.bd1xx.get_recid_from_ref')
+@mock.patch('inspirehep.dojson.hepnames.fields.bd1xx.get_record_ref')
+def test_experiments(mock_get_record_ref, mock_get_recid_from_ref, test_name,
+                     xml_snippet, expected_json, expected_marc):
     """Test if experiments is created correctly."""
-    assert (marcxml_to_json['experiments'][1]['name'] ==
-            json_to_marc['693'][1]['e'])
-    assert (marcxml_to_json['experiments'][1]['start_year'] ==
-            json_to_marc['693'][1]['s'])
-    assert (marcxml_to_json['experiments'][1]['end_year'] ==
-            json_to_marc['693'][1]['d'])
-    assert (marcxml_to_json['experiments'][1]['status'] ==
-            json_to_marc['693'][1]['z'])
+    mock_get_record_ref.side_effect = lambda x: x and 'mocked_recid_%s' % x
```

Comment:
this actually should not work... as you should be able to call get_record_ref with an extra parameter (the collection), will investigate